AdvanceTaskMarsPart1/Pages/ProfilePage.cs

```diff
@@ -0,0 +1,80 @@
+﻿using AdvanceTaskMarsPart1.Utilities;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace AdvanceTaskMarsPart1.Pages
+{
+    public class ProfilePage : CommonDriver
+    {
+        private IWebElement EditAvailabilityButton => driver.FindElement(By.XPath("//div[@class='four wide column']//div[@class='item']//strong[contains(text(),'Availability')]/../following-sibling::div//i[@class='right floated outline small write icon']"));
+        private IWebElement Availability           => driver.FindElement(By.XPath("//select[@name='availabiltyType']"));
+        private IWebElement successMessage         => driver.FindElement(By.XPath("//div[@class='ns-box-inner']"));
+        private Func<string, IWebElement> newAvailabilityType = availability => driver.FindElement(By.XPath($"//span[text()='{availability}']"));
+        private IWebElement CancelAvailabilityButton => driver.FindElement(By.XPath("//div[@class='four wide column']//div[@class='item']//strong[contains(text(),'Availability')]/../following-sibling::div//i[@class='remove icon']"));
+        private IWebElement EditHoursButton          => driver.FindElement(By.XPath("//div[@class='four wide column']//div[@class='item']//strong[contains(text(),'Hours')]/../following-sibling::div//i[@class='right floated outline small write icon']"));
+        private IWebElement Hours                    => driver.FindElement(By.XPath("//select[@name='availabiltyHour']"));
+        private Func<string, IWebElement> newHours   = hours => driver.FindElement(By.XPath($"//span[text()='{hours}']"));
+        private IWebElement EditEarnTargetButton     => driver.FindElement(By.XPath("//div[@class='four wide column']//div[@class='item']//strong[contains(text(),'Earn Target')]/../following-sibling::div//i[@class='right floated outline small write icon']"));
+        private IWebElement EarnTarget               => driver.FindElement(By.XPath("//select[@name='availabiltyTarget']"));
+        private Func<string, IWebElement> newEarnTarget = earnTarget => driver.FindElement(By.XPath($"//span[text()='{earnTarget}']"));
+
+        public void Edit_Availability(string availability)
+        {
+            //Wait.WaitToBeClickable("XPath", "//div[@class='four wide column']//div[@class='item']//strong[contains(text(),'Availability')]/../following-sibling::div//i[@class='right floated outline small write icon']", 6);
```

Comment:
Remove all commented out code it is bad practice to do this! Please check the whole code for this!